modules/jooby-yasson/README.md

```diff
@@ -0,0 +1,60 @@
+[![Maven](https://img.shields.io/maven-metadata/v/http/central.maven.org/maven2/org/jooby/jooby-yasson/maven-metadata.xml.svg)](http://mvnrepository.com/artifact/org.jooby/jooby-yasson/1.6.0)
```

Comment:
README.md at this level are automatically generated by the web site generator.

Move this file to `doc/doc/yasson/yasson.md` look at the [jackson.md](https://github.com/jooby-project/jooby/blob/master/doc/doc/jackson/jackson.md) as example